components/dotenv.rst

```diff
@@ -62,9 +62,52 @@ The ``load()`` method never overwrites existing environment variables. Use the
     // ...
     $dotenv->overload(__DIR__.'/.env');
 
+As you're working with the ``Dotenv`` component you'll notice that you might want
+to have different files depending on the environment you're working in. Typically
+this happens for local development or Continuous Integration where you might
+want to have different files for your ``test`` and ``dev`` environments.
+
+You can use ``Dotenv::loadEnv()`` to ease this process::
+
+    use Symfony\Component\Dotenv\Dotenv;
+
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv(__DIR__.'/.env');
+
+The ``Dotenv`` component will then look for the correct ``.env`` file to load
+in the following order::
+
+1. If ``.env`` exists, it is loaded first. In case there's no ``.env`` file but a
```

Comment:
In reStructured Text lists are created by `#.` (instead of using numbers)

Reply:
I would also add somewhere what "loaded first or last" means. Does the last one overwrite all values already set or does the first one get precedence?

Reply:
Ah, thx! All addressed :)